source/Classroom/Basics/Security/security_checklist.md

```diff
@@ -1,49 +1,36 @@
 ---
 seo:
   title: Security Checklist
-  description: Security Checklist
-  keywords: password, security, secure, virus, hack, script
+  description: Tips to help keeo your account secure.
+  keywords: password, security, secure, virus, hack, script, github
 title: Security Checklist
 weight: 0
 layout: page
-zendesk_id: 200182938
 navigation:
   show: true
 ---
 
-SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Take a few minutes, go through this checklist and make sure that your systems are secure.
+SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Take a few minutes and this checklist to help make ensure that your systems are secure.
 
-1)  Is your password robust?  (If your account is compromised, do a virus scan prior to changing your password in order to get rid of any key-logger's.)
+{% anchor h3 %} 1)  Use a robust password!  {% endanchor %}
 
-- Do you use the same password for multiple sites?
+- Don't use the same password for multiple sites, vary your passwords whenever possible.
+- If possible, use  upper and lower case letters, numerals, and special characters. 
+- Don’t use easily guessable words or phrases such as personal information, your company name, or abbreviations. 
+- Don't expose your password on Github!
 
-  - vary your passwords from site to site
+Consider setting up [Teammates]({{root_url}}/docs/User_Guide/Settings/teammates.html) to create, manage, and restrict additional sets of credentials to more precisily grant permissions within your SendGrid account.
```

Comment:
If you have more than one person accessing your SendGrid account, use [Teammates]({{root_url}}/docs/User_Guide/Settings/teammates.html) to manage additional sets of credentials to more precisely grant permissions within your SendGrid account.